docs/maui/converters/index-to-array-item-converter.md

```diff
@@ -0,0 +1,100 @@
+---
+title: IndexToArrayItemConverter - .NET MAUI Community Toolkit
+author: cliffagius
+description: "The IndexToArrayItemConverter is a converter that allows users to convert a int value binding to an item in an array. The int value being data bound represents the indexer used to access the array. The array is passed in through the ConverterParameter."
+ms.date: 04/14/2022
+---
+
+# IndexToArrayItemConverter
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `IndexToArrayItemConverter` is a converter that allows users to convert an `int` value binding to an item in an array. The `int` value being data bound represents the indexer used to access the array. The array is passed in through the `ConverterParameter`.
+
+## Syntax
+
+### XAML
+
+The `IndexToArrayItemConverter` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="MyLittleApp.MainPage">
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <toolkit:IndexToArrayItemConverter x:Key="IndexToArrayItemConverter" />
+            <x:Array x:Key="MyArray" Type="x:String">
+                <x:String>Value 1</x:String>
+                <x:String>Value 2</x:String>
+                <x:String>Value 3</x:String>
+                <x:String>Value 4</x:String>
+                <x:String>Value 5</x:String>
+            </x:Array>
+        </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <StackLayout>
+
+        <Label IsVisible="{Binding MyIntegerValue, Converter={StaticResource IndexToArrayItemConverter}, ConverterParameter={StaticResource MyArray}}" />
```

Comment:
I am not sure binding to `IsVisible` will provide anything sensible here, perhaps we bind to the `Text` property given our array items are strings?

```suggestion
        <Label Text="{Binding MyIntegerValue, Converter={StaticResource IndexToArrayItemConverter}, ConverterParameter={StaticResource MyArray}}" />
```

Reply:
I thought the same as I copied this from the XCT docs and wondered if it was some magic in the Converter but looking at the sample and api your correct so will make the change,